app/controllers/register/ConfirmationController.scala

```diff
@@ -41,8 +42,10 @@ class ConfirmationController @Inject()(
                                         getData: DraftIdRetrievalActionProvider,
                                         requireData: RegistrationDataRequiredAction,
                                         val controllerComponents: MessagesControllerComponents,
-                                        newIndividualView: newTrust.IndividualView,
-                                        newAgentView: newTrust.AgentView,
+                                        newTaxableIndividualView: newTrust.taxable.IndividualView,
+                                        newTaxableAgentView: newTrust.taxable.AgentView,
+                                        newIndividualView: IndividualView,
+                                        newAgentView: AgentView,
```

Comment:
@JoshuaH01  What is the difference between these views? I cannot tell from the names of the values or their imports.